stories/Menu.stories.svelte

```diff
@@ -0,0 +1,37 @@
+<script>
+  import { Meta, Template, Story } from "@storybook/addon-svelte-csf"
+  import { Menu } from "../components/mdc"
+
+  let content = 'Button slot'
+
+  const args = {
+    menuOpen: true,
```

Comment:
I think the piece you were missing is the toggleMenu handler, so with this you shouldn't need those styles.
```suggestion
    menuOpen: true,
    toggleMenu() {
      this.menuOpen = !menuOpen
    }

```